tracer/build/_build/PrepareRelease/IntegrationType.cs

```diff
@@ -0,0 +1,20 @@
+﻿// <copyright file="IntegrationGroups.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace PrepareRelease
+{
+    public enum IntegrationType
+    {
+        /// <summary>
+        /// Normal calltarget integration
+        /// </summary>
+        Normal = 0,
```

Comment:
[naming] Can we use `Default` instead? I see "normal" and it brings up all kind of thoughts in my head: What's "normal"? Are the other ones "abnormal"? etc...

(non-blocking since this is internal, same comment for the duplicate in `Datadog.Trace.ClrProfiler`)

Reply:
changed in 3490e0c